usaspending_api/references/views_v2/agency.py

```diff
@@ -71,10 +67,10 @@ def get(self, request, pk, format=None):
         response['results'] = {'agency_name': toptier_agency.name,
                                'active_fy': str(active_fiscal_year),
                                'active_fq': str(active_fiscal_quarter),
-                               'outlay_amount': str(submission.outlay_amount),
-                               'obligated_amount': str(submission.obligated_amount),
-                               'budget_authority_amount': str(submission.budget_authority_amount),
-                               'total_budget_authority_amount': total_budget_authority_amount
+                               'outlay_amount': str(aggregate_dict['outlay_amount']),
+                               'obligated_amount': str(aggregate_dict['obligated_amount']),
+                               'budget_authority_amount': str(aggregate_dict['budget_authority_amount']),
+                               'current_total_budget_authority_amount': total_budget_authority_amount
```

Comment:
Can you update the associated gist and put it in the PR description as a checklist item to be approved by frontend?